packages/@aws-cdk/aws-sns/lib/topic-base.ts

```diff
@@ -124,6 +125,13 @@ export abstract class TopicBase extends Resource implements ITopic {
     });
   }
 
+  public bind(_rule: notifications.INotificationRule): notifications.NotificationRuleTargetConfig {
+    return {
```

Comment:
Where's the `grantPublish` call that's present in the `SnsTopic` version of this target? Why is it missing here?

Reply:
Sorry, my mistake. I will fix and add test for this.